rgw/v2/tests/s3_swift/test_check_sharding_enabled.py

```diff
@@ -94,6 +94,35 @@ def test_exec(config, ssh_con):
         else:
             raise TestExecError("sharding has not enabled already")
 
+    if config.test_ops.get("zone_delete", False) is True:
+        log.info("Test zone deletion")
+        op = utils.exec_shell_cmd("radosgw-admin sync status")
+        lines = list(op.split("\n"))
+        for line in lines:
+            if "realm" in line:
+                realm = line[line.find("(") + 1 : line.find(")")]
+            if "zonegroup" in line:
+                zonegroup_name = line[line.find("(") + 1 : line.find(")")]
+                break
```

Comment:
@anrao19 , there is a function in utils to get the realm and source zone info https://github.com/red-hat-storage/ceph-qe-scripts/blob/master/rgw/v2/utils/utils.py#L636 , can we reuse it?

Reply:
Hi @mkasturi18 That method mentioned does not provide the zonegroup name 
source zone in this method is to check if its master or not